src/components/trending-developers/TrendingDevelopers.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState, useCallback } from 'react';
+import { useHistory } from 'react-router-dom';
+import { getTrendingDevelopers } from '../../services/github';
+import styles from './TrendingDevelopers.module.css';
+
+function TrendingDevelopers() {
+  const history = useHistory();
+  const [trendingDevelopers, setTrendingDevelopers] = useState();
+
+  useEffect(() => {
+    async function fetch() {
+      try {
+        const result = await getTrendingDevelopers();
+        setTrendingDevelopers(result);
+      } catch (err) {
+        // Fail silently
+        setTrendingDevelopers([]);
+      }
+    }
+
+    fetch();
+  }, []);
+
+  const goTo = useCallback((username) => (e) => {
+    e.preventDefault();
+    history.push(`/${username}`);
+  }, [history]);
+
+  return (
+    <div className={styles.trendingDevelopers}>
+      {trendingDevelopers && trendingDevelopers.length > 0 && (
```

Comment:
We might start using an util like: `isEmpty()`? In order to use it here like so: !isEmpty(trendingDevelopers)

Reply:
What about adding: https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining ?

Reply:
I like your `isEmpty` check; 
In regards to optional chaining and null coalescing operator, yeah, why not, but I'll leave this to you, maybe a PR? : )